examples/hyperbolic-multimodal/README.md

```
@@ -0,0 +1,178 @@
+This example demonstrates Hyperbolic's AI capabilities across four key areas:
+
+1. **Reasoning Models** - Advanced problem-solving and logical reasoning
+2. **Image Generation** - Text-to-image generation with multiple models
+3. **Audio Generation** - Text-to-speech synthesis
+4. **Multimodal Understanding** - Vision-language models for image analysis
+
+## Prerequisites
+
+1. Get your Hyperbolic API key from [https://app.hyperbolic.xyz](https://app.hyperbolic.xyz)
+2. Set the environment variable:
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)